assets/src/constants.js

```diff
@@ -51,6 +51,9 @@ export const ALLOWED_MOVABLE_BLOCKS = [
 	'core/verse',
 	'core/video',
 	'amp/amp-story-text',
+	'amp/amp-story-post-author',
+	'amp/amp-story-post-date',
+	'amp/amp-story-post-title',
```

Comment:
Should we register the `rounded` block style for these, too?

Reply:
I think so, makes it consistent. Will add it.